invisible_cities/cities/penthesilea.py

```diff
@@ -83,6 +82,14 @@ def penthesilea(files_in, file_out, compression, event_range, print_mod, detecto
     event_count_in  = df.spy_count()
     event_count_out = df.spy_count()
 
+    def append(l, e):
+        l.append(e)
+        return l
+
+    add_to_list = df.reduce(append, initial=[])
```

Comment:
I'd be tempted to

1. rename `add_to_list` to `collect_into_list`
2. contribute it to `dataflow` (If you do that, then rename `append` to `_append` when you put it into `dataflow`, or just use the `lambda` version; and, of course, add a documenting test.)